climada_petals/hazard/tc_rainfield.py

```diff
@@ -397,21 +397,13 @@ def from_tracks(
         TCRain
         """
         num_tracks = tracks.size
-        if centroids is None:
-            centroids = Centroids.from_base_grid(res_as=360, land=True)
-
-        if not centroids.coord.size:
-            centroids.set_meta_to_lat_lon()
-
```

Comment:
One could use the `climada.util.constants.NATEARTH_CENTROIDS` and still keep the centroids argument optional here

Reply:
gave it a try but found it unpractical as it covers the whole earth and causes computation downstream to last forever and consume all the memory.
Instead I took the boundaries from tracks and built a `Centroids` object spanning it. Now I wonder whether it should contain `on_land` coordinates only. But that could be done in a separate PR I suppose.

Reply:
I see. `on_land` wasn't used before, as far as I can see, so we can leave as is.